src/test/java/no/nav/veilarbaktivitet/db/dao/KafkaAktivitetDAOTest.java

```diff
@@ -0,0 +1,85 @@
+package no.nav.veilarbaktivitet.db.dao;
+
+import lombok.val;
+import no.nav.veilarbaktivitet.aktiviterTilKafka.KafkaAktivitetDAO;
+import no.nav.veilarbaktivitet.db.Database;
+import no.nav.veilarbaktivitet.db.DbTestUtils;
+import no.nav.veilarbaktivitet.domain.AktivitetData;
+import no.nav.veilarbaktivitet.domain.Person;
+import no.nav.veilarbaktivitet.mock.LocalH2Database;
+import no.nav.veilarbaktivitet.testutils.AktivitetDataTestBuilder;
+import org.junit.Before;
+import org.junit.Test;
+import org.springframework.jdbc.core.JdbcTemplate;
+
+import java.util.Date;
+import java.util.Optional;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.collection.IsCollectionWithSize.hasSize;
+import static org.junit.Assert.assertThat;
+
+public class KafkaAktivitetDAOTest {
+
+    private static final Person.AktorId AKTOR_ID = Person.aktorId("1234");
+
+    private final JdbcTemplate jdbcTemplate = LocalH2Database.getDb();
+    private final Database database = new Database(jdbcTemplate);
+    private final AktivitetDAO aktivitetDAO = new AktivitetDAO(database);
+    private final KafkaAktivitetDAO kafkaAktivitetDAO = new KafkaAktivitetDAO(database);
+
+
+    @Before
+    public void cleanUp(){
+        DbTestUtils.cleanupTestDb(jdbcTemplate);
+    }
+
+
+    @Test
+    public void opprette_og_hente_melding_som_ikke_er_sendt() {
```

Comment:
Er ikke så glad i tester i dette formatet med gherkin liknende oppbygning og navngivning med mange underscore fordi de ender opp med å ikke være så lesbare i etterkant og jeg syns det er vanskeligere å vurdere om man har god testdekning.

Foretrekker å navngi tester utifra metoden som testes sånn som dette:
<metode som testes>_<premiss>_<resultat>
for eksempel:
`hentOppTil5000MeldingerSomIkkeErSendt_AktivitetErIkkeSendt_returnererEnAktivitet`

Da blir det veldig tydelig at for å ha testdekning på metoden så må du også ha: 
`hentOppTil5000MeldingerSomIkkeErSendt_FlereAktivitetErIkkeSendt_returnererAlle`
`hentOppTil5000MeldingerSomIkkeErSendt_NoenAktiviteterErIkkeSendt_returnererUsendte`

Siden det er dere som eier området her så kan du jo bestemme litt selv hvor mange tester du vil ha for å evt spare tid på annen testing, men jeg vil gjerne at du renamer de testene du har ihvertfall så det er lett å se hva man har i etterkant og kan utvide.